ibis/backends/sql/compilers/pyspark.py

```diff
@@ -240,11 +240,27 @@ def visit_FirstValue(self, op, *, arg):
     def visit_LastValue(self, op, *, arg):
         return sge.IgnoreNulls(this=self.f.last(arg))
 
-    def visit_First(self, op, *, arg, where, order_by):
-        return sge.IgnoreNulls(this=self.agg.first(arg, where=where, order_by=order_by))
+    def visit_First(self, op, *, arg, where, order_by, include_null):
+        if where is not None and include_null:
+            raise com.UnsupportedOperationError(
+                "Combining `include_null=True` and `where` is not supported "
+                "by pyspark"
```

Comment:
Lotta boilerplate with the error handling, but we can clean that up later.

Reply:
Agreed, but I'd rather copy-paste stuff around than develop something that increases coupling. It's only a couple methods where this arg even makes sense.

Reply:
Yep, agreed.